.circleci/config.yml

```diff
@@ -967,7 +967,9 @@ jobs:
                     export USE_HERMES=0
                   fi
 
-                  cd packages/rn-tester && bundle exec pod install
+                  ls -l vendor/bundle/ruby
+                  ls -l vendor/bundle/ruby/3.1.0/bin
```

Comment:
Can we remove these lines?

Reply:
I'm still debugging the build as it does not work, looks like `bundle exec pod` fails because the `binstub` is not found for ruby `3.1.3`, it needs more investigation. This is not easy to debug as the whole build is extremely slow.

Reply:
Yeah, I know... We need to set aside some time to make it faster and more efficient... :(

Reply:
Thanks for the effort you are putting into it!

Reply:
It looks to me like the `test_ios_rntester` job is missing the `setup_ruby` step. This is where gems are installed, so Cocoapods doesn't exist. (I'm guessing maybe it was previously working because it was pulling the gems from a cache?)

Can we add the `setup_ruby` step to `test_ios_rntester`, or will that break something?